query.js

```diff
@@ -687,13 +687,13 @@ function parseExport(res) {
             index: ++activityIndex,
         }))
 
-        // occupations
-        if (d.military && d.military.length) d.occupations.forEach(a => activities.push({
+        // military
+        if (d.military && d.military.length) d.military.forEach(a => activities.push({
             ...entryBase,
-            lifeEvents: 'military careers',
+            lifeEvents: 'military career',
             eventsDetail1: a.officeType,
             eventsDetail2: a.rank,
-            place: a.place || "",
+            place: "",
```

Comment:
@ryanctan why is place always an empty string?

Reply:
There's no place variable in the military-careers entry field!